pkg/codegen/main.go

```diff
@@ -15,13 +15,15 @@ import (
 )
 
 func main() {
+	os.Unsetenv("GOPATH")
+
 	controllergen.Run(args.Options{
 		OutputPackage: "github.com/rancher/eks-operator/pkg/generated",
 		Boilerplate:   "pkg/codegen/boilerplate.go.txt",
 		Groups: map[string]args.Group{
 			"eks.cattle.io": {
 				Types: []interface{}{
-					v12.EKSClusterConfig{},
+					"./pkg/apis/eks.cattle.io/v1",
```

Comment:
Are changes to this file right ? Or is this debug ?

Reply:
This is the "some changes to support dapper." For some reason, codegen couldn't find the `v12.EKSClusterConfig` type when running codegen in dapper. Changing the type to the directory fixed this issue. Codegen will scan the directory.

I verified that these changes didn't change what was generated.

Reply:
Sounds good. Just keep in mind for GKE/AKS.